src/main/java/com/epam/aidial/core/controller/DeploymentPostController.java

```diff
@@ -340,8 +350,9 @@ private void handleProxyResponse(HttpClientResponse proxyResponse) {
             return;
         }
 
+        boolean isStreaming = context.getDeployment() instanceof Model && context.isStreamingRequest();
```

Comment:
Why only model?

Reply:
we need to collect last chunk for models only

Reply:
Why application streaming is so different?

Reply:
stats for applications are derived from model stats we don't need to wait for last chunk of the app response

Reply:
Sounds like a good comment at this place

Reply:
fixed